pkg/server/handler.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import "net/http"
+
+// StaticOKHandler returns a http handler which returns 200/ok on every call.
+func StaticOKHandler() http.Handler {
```

Comment:
Can't you use https://github.com/gardener/gardener/blob/master/pkg/healthz/default.go like in https://github.com/gardener/gardener/blob/master/cmd/gardener-controller-manager/app/gardener_controller_manager.go#L346?

Reply:
Yes, I can use it as an alternative. @timebertt and I agreed that for now a "static" handler is sufficient but we might want to improve it in the future. So I decided to add a simple static handler for that in order to make it obvious to developer that no health checks at all are executed. WDYT?

Reply:
Maybe I'm missing something, but I don't see a difference in the static handler compared to this default healthz checker, or?

Reply:
I'm thinking that we don't need no health checker with all its functionalities (start, set, stop, ...) if there are no real health checks happening. But I'm also happy to change it if it's beneficial.

Reply:
I've changed it in dfc526810fe87b11ea8a0b9f0593841558b71091. Your call :)